pages/plants/[id]/edit.js

```diff
@@ -21,7 +21,12 @@ export default function EditPage({ handleEditPlant }) {
     handleEditPlant(data, id, mutate);
   }
   if (status !== "authenticated") {
-    return <h1>Pech gehabt</h1>;
+    return (
+      <>
+        <BackArrow link={`/plants/${plant._id}`}></BackArrow>
```

Comment:
This can be a self closing tag if there are no children. 

```suggestion
        <BackArrow link={`/plants/${plant._id}`} />
```